installing/installing_azure/migrating-azure-dns-zones.adoc

```diff
@@ -0,0 +1,16 @@
+[id="migrating-azure-dns-zones"]
+= Migrating legacy Azure DNS zones
+include::modules/common-attributes.adoc[]
+:context: migrating-azure-dns-zones
+
+toc::[]
+
+If you installed an {product-title} version 4.2 or 4.3 cluster on Azure that uses installer-provisioned infrastructure, you must migrate the legacy private DNS zone to the new type of private DNS zone.
```

Comment:
This introductory content is repeated in the assembly and module. It shouldn't be an issue, but I wanted to make sure it was intentional.